browse/static/js/catalyzex.js

```diff
@@ -57,6 +72,14 @@
       .append(codeLink)
   } else {
     $output.append(`<p>No code found for this paper just yet.</p>`)
+
+    const { related_papers: relatedPapersWithCode } = await fetchRelatedPapersWithCode(paperTitle);
```

Comment:
I've just updated the PR to ensure we only show the related links when we're sure about their availability on our end. With this, we also don't need to slow down the normal CX API calls, especially in the cases where we do have code available and the related code info is irrelevant

Reply:
Thoughts on this approach VS. just conditionally processing and returning "related code" availability info via a CX API call, say based on the optional query param (hence not affecting normal CX api calls, while not making 2x API calls total either)